src/js/Index.js

```diff
@@ -29,9 +28,11 @@ import { DatePicker } from './Framework/DateTimePicker/DatePicker'
 import { DateTimePicker } from './Framework/DateTimePicker/DateTimePicker'
 import { TimePicker } from './Framework/DateTimePicker/TimePicker'
 import { Clipboard } from './Framework/Clipboard'
-import { ScrollEvent } from 'frameworkstylepackage/src/js/Framework/ScrollEvent'
-import { Theme } from 'frameworkstylepackage/src/js/Framework/Theme'
-import { FileInput } from 'frameworkstylepackage/src/js/Framework/FileInput'
+import { ScrollEvent } from './Framework/ScrollEvent'
+import { Theme } from './Framework/Theme'
+import { FileInput } from './Framework/FileInput'
+
+window.bootstrap = bootstrap
```

Comment:
Misschien een comment toevoegen waarom dit gebeurd? Iets als `// Expose Bootstrap so we can use it directly`

Reply:
@jonasdekeukelaere weet jij nog specifiek waarvoor dit nodig was?

Reply:
voor in andere imports aan bootstrap te kunnen, @Katrienvh gaat dat beter kunnen uitleggen